src/key.h

```diff
@@ -236,12 +236,6 @@ struct CExtKey {
     void SetSeed(Span<const std::byte> seed);
 };
 
-/** Initialize the elliptic curve support. May not be called twice without calling ECC_Stop first. */
-void ECC_Start();
```

Comment:
In commit "Refactor: Remove ECC_Start and ECC_Stop from key header" (65a205166b3c7d082a7528ea5036dff551d7eb6e)

It's probably good to remove these declarations, but I think it would be good to move the documentation to the cpp file so it is not lost. Notes like "May not be called twice" "No-op if ECC_Start wasn't called" seem useful to describe how these are intended to work.

Also references to these function in the ECC_Context comment below should be removed if they are private. Maybe replace "RAII wrapper for ECC_Start() and ECC_End(), initializing global ECC state" with "RAII class initializing and deinitializing global state for elliptic curve support."